libbeat/common/converter/converter.go

```diff
@@ -0,0 +1,205 @@
+package converter
```

Comment:
Could it be added to the current `schema` package so it can be used in current implementations without needing to refactor them?